src/http.ts

```diff
@@ -180,12 +180,13 @@ export async function getFilters(
   source: string,
   attempts = MAX_RETRY_ATTEMPTS,
   requestId?: string,
+  orgId?: string,
```

Comment:
What if org id is missing?

Reply:
If we need a route to the gateway, because we are on fedramp, it will throw an error. If not fedramp then nothing, it won't use org for the URL generation.